feature/src/main/resources/feature.xml

```diff
@@ -320,11 +320,6 @@ limitations under the License.
         <bundle>mvn:org.apache.jclouds.provider/glesys/${jclouds.version}</bundle>
     </feature>
 
-    <feature name='jclouds-hpcloud-compute' description='HP Cloud Compute' version='${project.version}' resolver='(obr)'>
-        <feature version='${project.version}'>jclouds-api-openstack-nova</feature>
-        <bundle>mvn:org.apache.jclouds.provider/hpcloud-compute/${jclouds.version}</bundle>
-    </feature>
```

Comment:
Would be nice to disentangle the removal of old providers with the addition of new ones.